main.go

```diff
@@ -112,3 +119,15 @@ Arguments:
 		helpTab,
 	)
 }
+
+//This section translates a windows path such as "C:\Users\user" to "\Users\user"
+//AND removes the default paths for mingw and git4windows to fix specifying a file path breaking
```

Comment:
Comments should have one space after the `//` and end in a period. They should also be hardwrapped at 80 characters. If you're using VS Code/code-server for this, you can set a ruler at column 80.

This doesn't read well, maybe try something like
```
This section translates a windows path such as "C:\Users\user" to "\Users\user" and removes the default paths for mingw and git4windows to issues when you specify a file path to start code-server in.

Reply:
Will implement in next push